_data/events_gdcr2018/Kiev.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "GDCR18 in Epam Kiev",
+  "url": "https://events.epam.com/events/coding-dojo-global-day-of-coderetreat",
```

Comment:
The link redirects to a login page for me. Is this a public event that anyone can join?

Reply:
Yes, but it seems that link is not opened for "the world" yet.

Reply:
In that case, I'd suggest we wait until the event page is available. We'll give you a shout out on twitter with the link as well, would be a pity if someone's interested but can't sign up away 😊

Reply:
Thanks, I'll have to communicate with our admins.